tests/protocols/test_websocket.py

```
@@ -1387,6 +1389,250 @@ async def app(scope: Scope, receive: ASGIReceiveCallable, send: ASGISendCallable
+
+    async def app(scope: Scope, receive: ASGIReceiveCallable, send: ASGISendCallable):
+        await receive()  # websocket.connect
+        await send({"type": "websocket.accept"})
+        accepted.set()
+        await send_requested.wait()
+        try:
+            await send({"type": "websocket.send", "text": "x"})
+        finally:
+            app_finished.set()
+
+    protocol, transport = connected_ws_protocol(app, ws_protocol_cls, http_protocol_cls)
```

Ruff (0.16.0)

[warning] Unpacked variable `transport` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)